image/templates/helm/stackrox-secured-cluster/values-public.yaml.example

```diff
@@ -293,7 +293,11 @@
 ## Collector specific configuration.
 #collector:
 #
-#  # Collection method to use. Can be one of:
+#  # If enabled, the value provided in `collectionMethod` is applied to the collector configuration.
+#  # Otherwise, the default collection method is used.
+#  forceCollectionMethod: false
```

Comment:
You'd also have to add this new field to `image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml` see this [doc](https://github.com/stackrox/stackrox/blob/bcbf1f34301bb8a6a13395a126713438828a03e3/image/templates/CHANGING_CHARTS.md#L23)

Reply:
I have added an entry for forceCollectionMethod